guides/v2.3/rest/modules/inventory/bulk-inventory.md

```diff
@@ -0,0 +1,137 @@
+
+Bulk transfer allows you to specify multiple products, the origin source from which to transfer inventory, and the destination source to receive quantities. The process moves the all product inventory from the origin source. You cannot transfer a partial quantity. However, Magento retains quantity information when you transfer stock.
+
+**Parameters**
+
+Name | Description | Type | Requirements
+--- | --- | --- | ---
+`skus` | A  comma-separated list of existing SKUs to transfer | Array | Required
+`originSource` | The original source of the SKUs | String | Required
+`destinationSource` | The target source for the SKUs. This source must be already defined. | String | Required
+`unassignFromOrigin` | If `true`, the original source is removed as a source for the products. If `false`, the original source is retained, but the products are marked as being out of stock. | Boolean | Required
+{:style="table-layout:auto;"}
+
+**Sample usage**
+
+`POST /V1/inventory/bulk-product-source-transfer`
+
+**Payload**
+
+```
+{
+  "skus": [
+    "testConfigProduct-red",
+    "testConfigProduct-blue"
+  ],
+  "originSource": "default",
+  "destinationSource": "central",
+  "unassignFromOrigin": true
+}
+```
+
+**Response**
+
+`true` if the request was successful
+
+## Bulk assign
+
+Use the `POST /V1/inventory/bulk-product-source-assign` endpoint to add one or more sources to your products. This endpoint helps when creating and assigning custom sources to your default or custom stocks and preparing new locations and inventory.
+
+After adding new custom sources, you can add inventory quantities per product or for multiple products using the `POST V1/inventory/source-items` endpoint. [Assign products to a source]({{ page.baseurl }}/rest/modules/inventory/manage-source-items.html#assign) describes this endpoint.
+
+The sources are added to the products with an inventory quantity of 0. You can add inventory amounts as available per source.
+
+**Parameters**
+
+Name | Description | Type | Requirements
+--- | --- | --- | ---
+`skus` | A comma-separated list of existing SKUs to assign | Array | Required
+`sourceCodes` | A comma-separated list of existing sources | Array | Required
+{:style="table-layout:auto;"}
+
+**Sample usage**
+
+`POST /V1/inventory/bulk-product-source-assign`
+
+**Payload**
+
+```
+{
+  "skus": [
+    "new-product3",
+    "new-product4"
+  ],
+  "sourceCodes": [
+    "central",
+    "east"
+  ]
+}
+```
+
+**Response**
+
+An ID that identifies the request, such as `1`.
+
+
+## Bulk unassign
+
+When unassigning a source from a product, you are indicating the product will no longer be stocked at that location. This process completely clears all inventory data for the source currently assigned to the product. If you need to move the existing inventory to a new location, consider using the bulk transfer endpoint (`POST /V1/inventory/bulk-product-source-transfer`).
```

Comment:
For inventory data, if you want some specifics, it would be quantity, in/out of stock, and notify quantity for that source.